inst/template/Makevars

```diff
@@ -0,0 +1,3 @@
+# -*- makefile -*-
+PKG_CXXFLAGS=-I$(PATH_DUST_INCLUDE) -DHAVE_INLINE $(SHLIB_OPENMP_CXXFLAGS)
```

Comment:
Are CFLAGS needed too in case we ever use openmp there too? Or are we unlikely to ever do that